assets/composables/settings.ts

```diff
@@ -69,7 +72,13 @@ const softWrap = computed({
   set: (value) => (settings.value.softWrap = value),
 });
 
+const collapseNav = computed({
```

Comment:
Ah I see, did you make this a setting? That's cool. Send that in a different PR.